src/components/avatar/AvatarBase.tsx

```diff
@@ -43,56 +44,10 @@ export type FullAvatarProps = BaseProps & {
 
 type AvatarBaseProps = BaseProps & {
   children?: ReactNode;
+  style?: React.CSSProperties;
```

Comment:
This is already part of the BaseProps, it's not necessary since you are not overwriting it.
```suggestion
```

Reply:
Ah! I thought I got them all. good catch